HW05/homework_5.py

```diff
@@ -0,0 +1,28 @@
+import time
+
+user_input = input("Введіть Ваш текст або число:\n")
+upper_text = 0
+lower_text = 0
+spaces = 0
+if user_input.isdigit():
```

Comment:
The task was to loop over every symbol from input string. This means that you should have one for-in loop and inside of it you should check every symbol